docs/mindsdb_sql/knowledge-bases.mdx

```diff
@@ -215,241 +452,294 @@ SELECT id, content, metadata, embeddings
 FROM <kb_name>_chromadb.storage_table;
 ```
 
-* KB with user-defined vector store (either ChromaDB or PGVector):
+* KB with user-defined vector store (either [PGVector](/integrations/vector-db-integrations/pgvector) or [ChromaDB](/integrations/vector-db-integrations/chromadb)):
 
 ```sql
 SELECT id, content, metadata, embeddings
 FROM <vector_store_connection_name>.<table_name>;
 ```
-</Accordion>
-
-<Accordion title="Examples">
 
-Here are the usage examples:
+### Example
 
-* Inserting into a knowledge base where the `content_columns` and `id_column` parameters were not specified:
+Here a sample knowledge base created in the previous **Example** section is inserted into.
 
 ```sql
 INSERT INTO my_kb
-SELECT book_id AS id, book_summary AS content 
-FROM my_books;
+    SELECT order_id, product, notes
+    FROM orders;
 ```
 
-* Inserting into a knowledge base where the `content_columns` and `id_column` parameters were specified as follows:
+<Note>
+When inserting into a knowledge base where the `content_columns` parameter was not specified, the column storing content must be aliased `AS content`.
```

Comment:
Maybe add alias in the example above

Reply:
There is an example for that in this Note below.